src/containers/student-home-page/find-tutor-block/find-tutor-block.styles.js

```diff
@@ -1,7 +1,21 @@
 export const styles = {
+  img: {
+    '@media (max-width: 900px)': {
+      display: 'none'
+    }
+  },
+  root: {
+    marginTop: '48px',
+    '@media (max-width: 900px)': {
+      marginTop: '24px'
+    }
```

Comment:
We don't use `@media`. Use breakpoints from MUI

Reply:
fixed